daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -365,14 +352,12 @@ object SBuiltin {
             // potentially very costly String to BigDecimal conversions. Take for example the String
             // "1." followed by millions of '0's
             val newString = s"$signPart$intPart.${Option(decPartOrNull).getOrElse("")}"
-            machine.returnValue(
-              SOptional(Some(SNumeric(Numeric.assertFromBigDecimal(scale, BigDecimal(newString))))),
-            )
+            SOptional(Some(SNumeric(Numeric.assertFromBigDecimal(scale, BigDecimal(newString)))))
           } else {
-            machine.returnValue(SV.None)
+            SV.None
```

Comment:
should this be: `machine.returnValue = SV.None` ?

Reply:
and some other places

Reply:
The `machine.returnValue = ` is in line 343.